src/lib/operators/maintenance/drop_table.cpp

```diff
@@ -0,0 +1,32 @@
+#include "drop_table.hpp"
+
+#include "storage/storage_manager.hpp"
+
+namespace opossum {
+
+DropTable::DropTable(const std::string& table_name)
+    : AbstractReadOnlyOperator(OperatorType::DropTable), table_name(table_name) {}
+
+const std::string DropTable::name() const { return "Drop Table"; }
+
+const std::string DropTable::description(DescriptionMode description_mode) const {
+  return name() + " '" + table_name + "'";
+}
+
+std::shared_ptr<const Table> DropTable::_on_execute() {
+  StorageManager::get().drop_table(table_name);
```

Comment:
This would set a precedent. So far we're not using AssertInput in operators and we had a long discussion about why we probably do not want to survive exceptions in Operators (Worker exception forwarding, etc). So I think the best we can do is some sort of pre-execution sanity checking that the console can trigger

Reply:
Ah, I forgot about that. I think we should have that sanity check, yes.

Reply:
Tracked in #1189 now